fe/user/src/pages/MapPage/Panel/VenueDetail/RestInfo/ShowInfo/ShowList.tsx

```diff
@@ -68,14 +81,15 @@ export const ShowList: React.FC<Props> = ({
               </StyledShowListItemTime>
             </StyledShowListItem>
           ))}
-        {showList && currentShowIndex !== -1 && (
-          <ShowDetail
-            showList={showList}
-            currentIndex={currentShowIndex}
-            onCloseClick={hideShowDetail}
-          />
-        )}
       </StyledShowListContent>
+
+      {showList.length > 0 && currentShowIndex !== -1 && (
```

Comment:
`showList.length`를 체크할 필요가 있나요?

Reply:
showList 초기값이 []이기 때문에 그것 자체의 평가값보다는 length 비교연산자로 평가하는게 더 명시적이고 의도를 추론하기 좋다고 생각했어요~ 😄

Reply:
`ShowDetail`에서 받는 `showList`가 빈배열이어도 정상적으로 동작할 것 같아서 체크할 필요가 없다고 생각돼서요

Reply:
```suggestion
      {showList && currentShowIndex !== -1 && (
```
빈 배열로 초기화하니까 이것도 괜찮을 것 같습니다.

Reply:
검사할 필요가 없다고 생각합니다. `showList`는 무조건 `[]`이기는 하니까요